feature-detects/scroll-to-options.js

```diff
@@ -0,0 +1,27 @@
+/*!
+{
+  "name": "scrollToOptions dictionary",
+  "property": "scrolltooptions"
+}
+!*/
+define(['Modernizr'], function (Modernizr) {
+  Modernizr.addTest('scrolltooptions', function () {
+    var returnTo = window.pageYOffset;
+    var needsFill = document.body.clientHeight <= window.innerHeight;
```

Comment:
so now use
    var body = getBody();
    var needsFill = body.clientHeight <= window.innerHeight;

(and for the other occurences of the body element too